.github/workflows/ci.yml

```diff
@@ -14,7 +14,7 @@ jobs:
     strategy:
```

Comment:
Since we'll release a new version of beanstalk-dispatch after this, you can update the changelog and __init__.py like we did in https://github.com/joshblum/beanstalk-dispatch/commit/eb802d8cd4c41df50226ce8952e079236db26fbc to bump the version. (It's otherwise an immediate commit right after this one.)